biotrainer/config/training_options.py

```diff
@@ -150,6 +195,12 @@ def required(self) -> bool:
 
 
 class PretrainedModel(TrainingOption, FileOption):
+    """
+    Configuration option for specifying a pretrained model.
+
+    This option allows users to provide a pretrained model file, typically in PyTorch (.pt) format,
```

Comment:
```suggestion
    This option allows users to provide a pretrained model file, typically in Safetensors (.safetensors) format,
```

Reply:
This format should also be changed in the allowed_formats property. .pt checkpoints are deprecated because they can lead to arbitrary code execution.